[BUILD] Can't build against TBB 2021

I can't build the 2.2.14.0 against TBB 2021.x, only with 2020_U2, because OpenImageIO can't find it, I think a little tweak need be done on [FindTBB.cmake](https://github.com/OpenImageIO/oiio/blob/master/src/cmake/modules/FindTBB.cmake) file.

I am using Slackware 15.0 beta
GCC 10.3.0
Cmake 3.20.2
Boost 1.76

I can't even run cmake when TBB 2021.2 is installed. I will put the output about that soon. I need recompile the package.
I have other problem too, when using OpenColorIO 2.0.0 I can't build OpenImageIO against OpenJPEG, the build break.

```
[ 63%] Building CXX object src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/__/jpeg2000.imageio/jpeg2000input.cpp.o
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:97:5: error: ‘opj_codec_t’ does not name a type; did you mean ‘opj_poc_t’?
   97 |     opj_codec_t* m_codec;
      |     ^~~~~~~~~~~
      |     opj_poc_t
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:98:5: error: ‘opj_stream_t’ does not name a type
   98 |     opj_stream_t* m_stream;
      |     ^~~~~~~~~~~~
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:105:5: error: ‘opj_codec_t’ does not name a type; did you mean ‘opj_poc_t’?
  105 |     opj_codec_t* create_decompressor();
      |     ^~~~~~~~~~~
      |     opj_poc_t
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:143:26: error: ‘opj_codec_t’ has not been declared
  143 |     void setup_event_mgr(opj_codec_t* codec)
      |                          ^~~~~~~~~~~
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp: In member function ‘void OpenImageIO_v2_2::Jpeg2000Input::destroy_stream()’:
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:110:13: error: ‘m_stream’ was not declared in this scope; did you mean ‘ofstream’?
  110 |         if (m_stream) {
      |             ^~~~~~~~
      |             ofstream
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:111:13: error: ‘opj_stream_destroy’ was not declared in this scope; did you mean ‘opj_image_destroy’?
  111 |             opj_stream_destroy(m_stream);
      |             ^~~~~~~~~~~~~~~~~~
      |             opj_image_destroy
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp: In member function ‘void OpenImageIO_v2_2::Jpeg2000Input::setup_event_mgr(int*)’:
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:145:9: error: ‘opj_set_error_handler’ was not declared in this scope
  145 |         opj_set_error_handler(codec, openjpeg_error_callback, this);
      |         ^~~~~~~~~~~~~~~~~~~~~
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:146:9: error: ‘opj_set_warning_handler’ was not declared in this scope
  146 |         opj_set_warning_handler(codec, openjpeg_dummy_callback, NULL);
      |         ^~~~~~~~~~~~~~~~~~~~~~~
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:147:9: error: ‘opj_set_info_handler’ was not declared in this scope
  147 |         opj_set_info_handler(codec, openjpeg_dummy_callback, NULL);
      |         ^~~~~~~~~~~~~~~~~~~~
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp: In member function ‘void OpenImageIO_v2_2::Jpeg2000Input::init()’:
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:177:5: error: ‘m_codec’ was not declared in this scope
  177 |     m_codec                   = NULL;
      |     ^~~~~~~
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:178:5: error: ‘m_stream’ was not declared in this scope; did you mean ‘ofstream’?
  178 |     m_stream                  = NULL;
      |     ^~~~~~~~
      |     ofstream
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp: In member function ‘virtual bool OpenImageIO_v2_2::Jpeg2000Input::open(const string&, OpenImageIO_v2_2::ImageSpec&)’:
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:192:5: error: ‘m_codec’ was not declared in this scope
  192 |     m_codec = create_decompressor();
      |     ^~~~~~~
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:192:15: error: ‘create_decompressor’ was not declared in this scope; did you mean ‘destroy_decompressor’?
  192 |     m_codec = create_decompressor();
      |               ^~~~~~~~~~~~~~~~~~~
      |               destroy_decompressor
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:211:5: error: ‘m_stream’ was not declared in this scope; did you mean ‘ofstream’?
  211 |     m_stream = opj_stream_create_default_file_stream(m_file, true);
      |     ^~~~~~~~
      |     ofstream
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:211:16: error: ‘opj_stream_create_default_file_stream’ was not declared in this scope
  211 |     m_stream = opj_stream_create_default_file_stream(m_file, true);
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:220:10: error: ‘opj_read_header’ was not declared in this scope
  220 |     if (!opj_read_header(m_stream, m_codec, &m_image)) {
      |          ^~~~~~~~~~~~~~~
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:246:56: error: no matching function for call to ‘max(const int&, unsigned int&)’
  246 |         maxPrecision = std::max(comp.prec, maxPrecision);
      |                                                        ^
In file included from /usr/include/c++/10.3.0/vector:60,
                 from /tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:6:
/usr/include/c++/10.3.0/bits/stl_algobase.h:254:5: note: candidate: ‘template<class _Tp> constexpr const _Tp& std::max(const _Tp&, const _Tp&)’
  254 |     max(const _Tp& __a, const _Tp& __b)
      |     ^~~
/usr/include/c++/10.3.0/bits/stl_algobase.h:254:5: note:   template argument deduction/substitution failed:
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:246:56: note:   deduced conflicting types for parameter ‘const _Tp’ (‘int’ and ‘unsigned int’)
  246 |         maxPrecision = std::max(comp.prec, maxPrecision);
      |                                                        ^
In file included from /usr/include/c++/10.3.0/vector:60,
                 from /tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:6:
/usr/include/c++/10.3.0/bits/stl_algobase.h:300:5: note: candidate: ‘template<class _Tp, class _Compare> constexpr const _Tp& std::max(const _Tp&, const _Tp&, _Compare)’
  300 |     max(const _Tp& __a, const _Tp& __b, _Compare __comp)
      |     ^~~
/usr/include/c++/10.3.0/bits/stl_algobase.h:300:5: note:   template argument deduction/substitution failed:
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:246:56: note:   deduced conflicting types for parameter ‘const _Tp’ (‘int’ and ‘unsigned int’)
  246 |         maxPrecision = std::max(comp.prec, maxPrecision);
      |                                                        ^
In file included from /usr/include/c++/10.3.0/algorithm:62,
                 from /tmp/SBo/oiio-Release-2.2.14.0/src/include/OpenImageIO/string_view.h:9,
                 from /tmp/SBo/oiio-Release-2.2.14.0/src/include/OpenImageIO/filesystem.h:32,
                 from /tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:10:
/usr/include/c++/10.3.0/bits/stl_algo.h:3485:5: note: candidate: ‘template<class _Tp> constexpr _Tp std::max(std::initializer_list<_Tp>)’
 3485 |     max(initializer_list<_Tp> __l)
      |     ^~~
/usr/include/c++/10.3.0/bits/stl_algo.h:3485:5: note:   template argument deduction/substitution failed:
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:246:56: note:   mismatched types ‘std::initializer_list<_Tp>’ and ‘int’
  246 |         maxPrecision = std::max(comp.prec, maxPrecision);
      |                                                        ^
In file included from /usr/include/c++/10.3.0/algorithm:62,
                 from /tmp/SBo/oiio-Release-2.2.14.0/src/include/OpenImageIO/string_view.h:9,
                 from /tmp/SBo/oiio-Release-2.2.14.0/src/include/OpenImageIO/filesystem.h:32,
                 from /tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:10:
/usr/include/c++/10.3.0/bits/stl_algo.h:3491:5: note: candidate: ‘template<class _Tp, class _Compare> constexpr _Tp std::max(std::initializer_list<_Tp>, _Compare)’
 3491 |     max(initializer_list<_Tp> __l, _Compare __comp)
      |     ^~~
/usr/include/c++/10.3.0/bits/stl_algo.h:3491:5: note:   template argument deduction/substitution failed:
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:246:56: note:   mismatched types ‘std::initializer_list<_Tp>’ and ‘int’
  246 |         maxPrecision = std::max(comp.prec, maxPrecision);
      |                                                        ^
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp: At global scope:
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:369:1: error: ‘opj_codec_t’ does not name a type; did you mean ‘opj_poc_t’?
  369 | opj_codec_t*
      | ^~~~~~~~~~~
      | opj_poc_t
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp: In member function ‘void OpenImageIO_v2_2::Jpeg2000Input::destroy_decompressor()’:
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:392:9: error: ‘m_codec’ was not declared in this scope
  392 |     if (m_codec) {
      |         ^~~~~~~
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:393:9: error: ‘opj_destroy_codec’ was not declared in this scope; did you mean ‘opj_destroy_compress’?
  393 |         opj_destroy_codec(m_codec);
      |         ^~~~~~~~~~~~~~~~~
      |         opj_destroy_compress
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp: In member function ‘void OpenImageIO_v2_2::Jpeg2000Input::read_scanline(int, int, void*)’:
/tmp/SBo/oiio-Release-2.2.14.0/src/jpeg2000.imageio/jpeg2000input.cpp:426:33: error: ‘OPJ_CLRSPC_SYCC’ was not declared in this scope; did you mean ‘CLRSPC_SYCC’?
  426 |     if (m_image->color_space == OPJ_CLRSPC_SYCC)
      |                                 ^~~~~~~~~~~~~~~
      |                                 CLRSPC_SYCC
make[2]: *** [src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/build.make:1476: src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/__/jpeg2000.imageio/jpeg2000input.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:1764: src/libOpenImageIO/CMakeFiles/OpenImageIO.dir/all] Error 2
make: *** [Makefile:166: all] Error 2
```

```
mkdir -p build
cd build
  cmake \
    -DCMAKE_C_FLAGS:STRING="-O2 -fPIC -D__STDC_CONSTANT_MACROS" \
    -DCMAKE_CXX_FLAGS:STRING="-O2 -fPIC -D__STDC_CONSTANT_MACROS -std=c++11 -fcommon" \
    -DOPENJPEG_INCLUDE_DIR="/usr/include/openmj2-2.1" \
    -DOPENJPEG_OPENJPEG_LIBRARY_RELEASE="/usr/lib64/libopenmj2.so" \
    -DCMAKE_BUILD_TYPE=Release \
    -DCMAKE_INSTALL_PREFIX=/usr \
    -DPYLIB_INCLUDE_SONAME:BOOL=TRUE \
    -DPYLIB_LIB_PREFIX:BOOL=FALSE \
    -DUSE_OPENJPEG:BOOL=TRUE \
    -DOIIO_BUILD_TESTS:BOOL=FALSE \
    -DUSE_PYTHON:BOOL=TRUE \
    -DPYTHON_VERSION=$(python3 --version | cut -d' ' -f2) \
    ..
  make
```


```
-- The CXX compiler identification is GNU 10.3.0
-- The C compiler identification is GNU 10.3.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/local/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/local/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Configuring OpenImageIO 2.2.14.0
-- CMake 3.20.2
-- CMake system           = Linux-5.12.0
-- CMake system name      = Linux
-- Project source dir     = /tmp/SBo/oiio-Release-2.2.14.0
-- Project build dir      = /tmp/SBo/oiio-Release-2.2.14.0/build
-- Project install prefix = /usr
-- Configuration types    = 
-- Build type             = Release
-- Setting Namespace to: OpenImageIO_v2_2
-- CMAKE_CXX_COMPILER     = /usr/local/bin/c++
-- CMAKE_CXX_COMPILER_ID  = GNU
-- Building for C++11
-- clang-format found: /usr/bin/clang-format
-- 
-- * Checking for dependencies...
-- *   - Missing a dependency 'Package'?
-- *     Try cmake -DPackage_ROOT=path or set environment var Package_ROOT=path
-- *     For many dependencies, we supply src/build-scripts/build_Package.bash
-- *   - To exclude an optional dependency (even if found),
-- *     -DUSE_Package=OFF or set environment var USE_Package=OFF 
-- 
CMake Warning at /usr/share/cmake-3.20/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:1476 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:2086 (_Boost_MISSING_DEPENDENCIES)
  src/cmake/checked_find_package.cmake:89 (find_package)
  src/cmake/externalpackages.cmake:60 (checked_find_package)
  CMakeLists.txt:121 (include)


CMake Warning at /usr/share/cmake-3.20/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:1476 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:2086 (_Boost_MISSING_DEPENDENCIES)
  src/cmake/checked_find_package.cmake:89 (find_package)
  src/cmake/externalpackages.cmake:60 (checked_find_package)
  CMakeLists.txt:121 (include)


CMake Warning at /usr/share/cmake-3.20/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:1476 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:2086 (_Boost_MISSING_DEPENDENCIES)
  src/cmake/checked_find_package.cmake:89 (find_package)
  src/cmake/externalpackages.cmake:60 (checked_find_package)
  CMakeLists.txt:121 (include)


CMake Warning at /usr/share/cmake-3.20/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:1476 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:2086 (_Boost_MISSING_DEPENDENCIES)
  src/cmake/checked_find_package.cmake:89 (find_package)
  src/cmake/externalpackages.cmake:60 (checked_find_package)
  CMakeLists.txt:121 (include)


CMake Warning at /usr/share/cmake-3.20/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:1476 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:2086 (_Boost_MISSING_DEPENDENCIES)
  src/cmake/checked_find_package.cmake:89 (find_package)
  src/cmake/externalpackages.cmake:60 (checked_find_package)
  CMakeLists.txt:121 (include)


CMake Warning at /usr/share/cmake-3.20/Modules/FindBoost.cmake:1354 (message):
  New Boost version may have incorrect or missing dependencies and imported
  targets
Call Stack (most recent call first):
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:1476 (_Boost_COMPONENT_DEPENDENCIES)
  /usr/share/cmake-3.20/Modules/FindBoost.cmake:2086 (_Boost_MISSING_DEPENDENCIES)
  src/cmake/checked_find_package.cmake:89 (find_package)
  src/cmake/externalpackages.cmake:60 (checked_find_package)
  CMakeLists.txt:121 (include)


-- Found Boost 107600 
-- Found ZLIB 1.2.11 
-- Found TIFF 4.3.0 
-- Could NOT find Imath (missing: Imath_DIR)
-- Found OpenEXR 2.5.5 
CMake Warning (dev) at /usr/share/cmake-3.20/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (JPEG) does
  not match the name of the calling package (JPEGTurbo).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  src/cmake/modules/FindJPEGTurbo.cmake:36 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  src/cmake/checked_find_package.cmake:89 (find_package)
  src/cmake/externalpackages.cmake:119 (checked_find_package)
  CMakeLists.txt:121 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found JPEG: /usr/lib64/libjpeg.so  
-- Found JPEGTurbo  
-- Using internal PugiXML
-- Found Python 3.9.4 
-- Found PNG 1.6.37 
-- Found BZip2 1.0.8 
-- Found Freetype 2.10.4 
-- Found HDF5 1.12.0 
-- Found OpenColorIO 2.0.0 
-- Found OpenCV 4.5.2 
CMake Error at src/cmake/modules/FindTBB.cmake:197 (file):
  file failed to open for reading (No such file or directory):

    /usr/include/tbb/tbb_stddef.h
Call Stack (most recent call first):
  src/cmake/checked_find_package.cmake:89 (find_package)
  src/cmake/externalpackages.cmake:166 (checked_find_package)
  CMakeLists.txt:121 (include)


-- TBB library not found 
--     Try setting TBB_ROOT ? 
-- Found DCMTK 3.6.6 
-- Found FFmpeg  
-- Field3D library not found 
--     Try setting Field3D_ROOT ? 
-- Found GIF 5.2.1 
-- Libheif library not found 
--     Try setting Libheif_ROOT ? 
-- Found LibRaw 0.20.2 
-- Found OpenJpeg 1.5 
CMake Warning (dev) at src/cmake/checked_find_package.cmake:77 (set):
  Cannot set "ENABLE_OpenVDB": current scope has no parent.
Call Stack (most recent call first):
  src/cmake/externalpackages.cmake:195 (checked_find_package)
  CMakeLists.txt:121 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Not using OpenVDB -- disabled (because TBB was not found) 
-- PTex library not found 
--     Try setting PTex_ROOT ? 
-- Found WebP  
-- Not using R3DSDK -- disabled  
-- Nuke library not found 
--     Try setting Nuke_ROOT ? 
-- Found OpenGL  
-- Found Qt5 5.15.2 
-- Libsquish library not found 
--     Try setting Libsquish_ROOT ? 
-- Found fmt 70103 
-- Found Robinmap  
CMake Warning at src/heif.imageio/CMakeLists.txt:10 (message):
  heif plugin will not be built


-- Found pybind11 2.6.2 
-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests gpsread;oiiotool;oiiotool-attribs;oiiotool-copy;oiiotool-xform;maketx;oiiotool-maketx;misnamed-file;texture-crop;texture-cropover;texture-filtersize;texture-overscan;texture-wrapfill;texture-res;texture-maxres
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests python-typedesc;python-paramlist;python-imagespec;python-roi;python-deep;python-colorconfig;python-imageinput;python-imageoutput;python-imagebuf;python-imagebufalgo
--   -> You can find it at 

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../bmpsuite
--   -> Will not run tests bmp
--   -> You can find it at http://entropymine.com/jason/bmpsuite/bmpsuite.zip

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests dpx
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../fits-images
--   -> Will not run tests fits
--   -> You can find it at http://www.cv.nrao.edu/fits/data/tests/

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests gif
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests ico
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests iff
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../j2kp4files_v1_5
--   -> Will not run tests jpeg2000
--   -> You can find it at http://www.itu.int/net/ITU-T/sigdb/speimage/ImageForm-s.aspx?val=10100803

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../openexr-images
--   -> Will not run tests openexr-suite;openexr-multires;openexr-chroma;openexr-v2;openexr-window;perchannel;oiiotool-deep
--   -> You can find it at http://www.openexr.com/downloads.html

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../openexr-images
--   -> Will not run tests openexr-damaged
--   -> You can find it at http://www.openexr.com/downloads.html

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests png
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests pnm
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests psd;psd-colormodes
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images/raw
--   -> Will not run tests raw
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests rla
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests sgi
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests targa-tgautils
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../libtiffpic
--   -> Will not run tests tiff-suite;tiff-depths;tiff-misc
--   -> You can find it at http://www.simplesystems.org/libtiff/images.html

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images/webp
--   -> Will not run tests webp
--   -> You can find it at Recent checkout of oiio-images

-- 

Did not find /tmp/SBo/oiio-Release-2.2.14.0/../oiio-images
--   -> Will not run tests zfile
--   -> You can find it at Recent checkout of oiio-images

-- Configuring incomplete, errors occurred!
```

--- a/INSTALL.md
+++ b/INSTALL.md
@@ -42,7 +42,7 @@ NEW or CHANGED MINIMUM dependencies since the last major release are **bold**.
      * Field3D (tested through 1.7.3)
  * If you want support for OpenVDB files:
      * OpenVDB >= 5.0 (tested through 8.0) and Intel TBB >= 2018 (tested
-       through 2020_U3)
+       through 2021)
      * Note that OpenVDB 8.0+ requires C++14 or higher.
  * If you want support for converting to and from OpenCV data structures,
    or for capturing images from a camera:

--- a/src/cmake/externalpackages.cmake
+++ b/src/cmake/externalpackages.cmake
@@ -161,8 +161,8 @@ checked_find_package (OpenCV
 # Intel TBB
 set (TBB_USE_DEBUG_BUILD OFF)
 checked_find_package (TBB 2017
-                   DEFINITIONS  -DUSE_TBB=1
-                   ISDEPOF      OpenVDB)
+                      DEFINITIONS  -DUSE_TBB=1
+                      PREFER_CONFIG)
 
 checked_find_package (DCMTK VERSION_MIN 3.6.1)  # For DICOM images
 checked_find_package (FFmpeg VERSION_MIN 2.6)

--- a/src/cmake/modules/FindTBB.cmake
+++ b/src/cmake/modules/FindTBB.cmake
@@ -134,7 +134,7 @@ if(NOT TBB_FOUND)
   # Find the TBB include dir
   ##################################
   
-  find_path(TBB_INCLUDE_DIRS tbb/tbb.h
+  find_path(TBB_INCLUDE_DIRS tbb/tbb_stddef.h
       HINTS ${TBB_INCLUDE_DIR} ${TBB_SEARCH_DIR}
       PATHS ${TBB_DEFAULT_SEARCH_DIR})
   

